packages/ckeditor5-font-mapper/src/fontMapping/index-doc.ts

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
We should review, if we want this extra `fontMapping` directory. In general, we (and others) prefer having flat hierarchies (exceptions like for itests, where grouping eases lookup of relevant files is one of the exceptions).